lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -37,7 +37,7 @@
 
 
 def meraki_argument_spec():
-    return dict(auth_key=dict(type='str', no_log=True, fallback=(env_fallback, ['MERAKI_KEY'])),
+    return dict(auth_key=dict(type='str', no_log=True, fallback=(env_fallback, ['MERAKI_KEY']), required=True),
```

Comment:
You should also update `plugins/doc_fragments/meraki.py` so that the documentation will reflect the change.

Reply:
Updated. Shippable is working on it.